web/components/source-item.js

```diff
@@ -113,19 +113,25 @@ template.innerHTML = `
 `;
 
 const addrString = (addr) => {
-	if (!addr) {
+	if (!addr || !addr.value) {
 		return "Unknown address";
 	}
 
+	console.log(addr.value);
 	const val = bufToAddressString(addr.value.addr);
+	const addressType = addr.value.type;
 
-	if (addr.type === BT_DataType.BT_DATA_RPA) {
-		return `${val} (random)`;
-	} else if (addr.type === BT_DataType.BT_DATA_IDENTITY) {
+	if (addressType === 0) {
```

Comment:
You don't have any values defined in the `BT_DataType.BT_DATA_*`?

Reply:
not for this - maybe we should, but needs to be aligned with the firmware (next PR ;))